values.yaml

```diff
@@ -232,6 +232,7 @@ persistence:
       accesskeysecret: accesskeysecret
       region: regionname
       bucket: bucketname
+      ### endpoint format: [bucket].[region].aliyuncs.com or [bucket].[region]-internal.aliyuncs.com
```

Comment:
```
An endpoint which defaults to [bucket].[region].aliyuncs.com or [bucket].[region]-internal.aliyuncs.com (when internal=true). You can change the default endpoint by changing this value.
```
Per the doc , the format `[bucket].[region].aliyuncs.com` or `[bucket].[region]-internal.aliyuncs.com` is the default value of endpoint. So this isn't the required format for the customized endpoint?